src/app/test/page.tsx

```
@@ -0,0 +1,632 @@
+    y1="0.398437"
+    x2="4.77268"
+    y2="9.08306"
+    gradientUnits="userSpaceOnUse"
+  >
+    <stop stopColor="#4C3876" />
+    <stop offset="1" stopColor="#475E99" />
+  </linearGradient>
+);
+
+const page = () => {
+  const [isPress, setIsPress] = useState(false);
```

ESLint

[error] React Hook "useState" is called in function "page" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)